program-payy-inclusion/src/main.rs

```diff
@@ -0,0 +1,49 @@
+#![doc = include_str!("../README.md")]
+#![no_main]
+
+sp1_zkvm::entrypoint!(main);
+use celestia_types::{blob::Blob, hash::Hash, AppVersion, ShareProof};
+use eq_common::{PayyInclusionToDataRootProofInput, PayyInclusionToDataRootProofOutput};
+use sha3::{Digest, Keccak256};
+use sp1_bn254_poseidon::fields::bn256::FpBN256;
+
+pub fn main() {
+    println!("cycle-tracker-start: deserialize input");
+    let input: PayyInclusionToDataRootProofInput = sp1_zkvm::io::read();
+    println!("cycle-tracker-end: deserialize input");
+
+    println!("cycle-tracker-start: create blob");
+    let blob =
+        Blob::new(input.namespace_id, input.data, AppVersion::V3).expect("Failed creating blob");
```

Comment:
```suggestion
        Blob::new(input.namespace_id, input.data, AppVersion::V4).expect("Failed creating blob");
```
need to test against latest node versions.